.github/workflows/rust.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         uses: actions/cache@v1
         with:
           path: .docker
-          key: debian-10-rust-${{ hashFiles('rust/**.rs') }}
+          key: debian-10-rust-${{ hashFiles('rust/**/**.rs', 'rust/**/Cargo.toml') }}
```

Comment:
According to [GHA docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) `**` should match nested directories as well.

Reply:
Without this, `hashFiles` becomes an empty string.